perllib/mySociety/EmailUtil.pm

```diff
@@ -84,6 +84,30 @@ sub is_valid_email ($) {
 }
 
 
+=item is_valid_email_list ADDRESSES
+
+Takes a string containing one or more comma-separated email addresses and calls
+is_valid_email on each, returning success iff they're all valid.
+
+=cut
+
+sub is_valid_email_list ($) {
+    my $addresses = shift;
+
+    my @emails;
+    if ( $addresses =~ /,/ ) {
+        @emails = split(/,/, $addresses);
+    } else {
+        @emails = ( $addresses );
+    }
```

Comment:
I think this can all be `my @emails = split(/,/, $addresses);` - if there's no `,`, the split will return the string as you'd expect.